README.md

```diff
@@ -127,8 +127,8 @@ rspec spec
 
```

Comment:
**[OPTIONAL]**
- [ ] It would be nice if you make your pull request summary more detailed to make your pull request more professional and descriptive 👍